### PR TITLE
ENCD-3616-donor-parents

### PR DESCRIPTION
**Pull request by keenangraham**
Submitting for Idan. Adds parent_strains to fly and worm donor (already in mouse donor) and limits these to maximum of two strains from same organism.

### CODE REVIEW

File: src/encoded/types/donor.py [hunk 1]
```diff
@@ -66,7 +66,10 @@ def characterizations(self, request, characterizations):
 class MouseDonor(Donor):
     item_type = 'mouse_donor'
     schema = load_schema('encoded:schemas/mouse_donor.json')
-    embedded = Donor.embedded + ['references']
+    embedded = Donor.embedded + ['references',
+                                 'genetic_modifications',
+                                 'genetic_modifications.modified_site_by_target_id',
+                                 'genetic_modifications.treatments', ]
```
[the remark this answers is not in the document]
Comment (hitz) [reply src/encoded/types/donor.py]: yeah I agree with this, but there is no hurry

File: src/encoded/schemas/donor.json [hunk 2]
```diff
@@ -20,21 +20,6 @@
         { "$ref": "mixins.json#/uuid" },
         { "$ref": "mixins.json#/schema_version" }
     ],
-    "dependencies": {
```
[the remark this answers is not in the document]
Comment (hitz) [reply src/encoded/schemas/donor.json]: @caseylitton can you resubmit this PR without this change?

[the remark this answers is not in the document]
Comment (keenangraham) [reply src/encoded/schemas/donor.json]: Leaving for now.

File: src/encoded/upgrade/donor.py [hunk 3]
```diff
@@ -208,3 +208,11 @@ def human_donor_10_11(value, system):
     # https://encodedcc.atlassian.net/browse/ENCD-3536
     if 'genetic_modifications' in value:
         value.pop('genetic_modifications', None)
+
```
[the remark this answers is not in the document]
Comment (keenangraham) [reply src/encoded/upgrade/donor.py]: Added.

File: src/encoded/tests/test_schema_donor.py [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+
+
+@pytest.fixture
+def mouse_donor_to_test(testapp, lab, award, mouse):
+    return {
+        'award': award['@id'],
+        'lab': lab['@id'],
+        'organism': mouse['@id'],
+    }
+
```
[the remark this answers is not in the document]
Comment (keenangraham) [reply src/encoded/tests/test_schema_donor.py]: Added.

File: src/encoded/types/donor.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (keenangraham) [reply src/encoded/types/donor.py]: Removed comma but multiple instances of this formatting in file.

**Comment by hitz** [comment]
for @keenangraham I don't think you have submitted a change, so just hold off until we confirm that this is OK.

**Pull request merged**